common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -22,6 +25,19 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
   val soundcloudUrlV1 = "http://api.soundcloud.com%2Ftracks%2F1234"
   val soundcloudUrlV2 = "https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/1234"
   val soundcloudUrlNoTrackId = "https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/foobar"
+  val commentAvatarClass = Some("d2-avatar")
```

Comment:
@NataliaLKB Thanks, thats a good point! No real need for Options here at all.
I've removed the "Some" from these in the latest commit.